### PR TITLE
Readme refine

### PR DESCRIPTION
**Pull request by simonsmh**
1. update libsodium and mbedtls version
2. remove some duplicate information
3. move some building suggestion to other part of document

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -169,13 +138,14 @@ sudo systemctl start shadowsocks-libev      # for systemd
 
 ### Fedora & RHEL
 
-Supported distributions include
-- Fedora 22, 23, 24
-- RHEL 6, 7 and derivatives (including CentOS, Scientific Linux)
+Supported Platforms:
```
Comment (madeye) [inline README.md]: They are not "platforms", but "distributions".

File: README.md [hunk 2]
```diff
@@ -300,8 +296,7 @@ brew install shadowsocks-libev
 
 ## Usage
 
-For a detailed and complete list of all supported arguments, you may refer to the
-man pages of the applications, respectively.
+For a detailed and complete list of all supported arguments, you may refer to the man pages of the applications, respectively.
```
Comment (madeye) [inline README.md]: No need to merge two lines together.

File: README.md [hunk 3]
```diff
@@ -93,16 +68,18 @@ try `configure --help`.
 
 #### Install from repository
 
-**Note: The repositories doesn't always contain the latest version. Please build from source if you want the latest version (see below)**
+**Note: The repositories doesn't always contain the latest version. Please build from source if you want the latest version. (see below)**
 
-Shadowsocks-libev is available in the official repository for Debian 9("Stretch"), unstable, Ubuntu 16.10 and later derivatives:
+Shadowsocks-libev is available in the official repository for **Debian 9, Ubuntu 16.10** and later derivatives:
```
Comment (madeye) [inline README.md]: "unstable" is missing.

**Comment by simonsmh** [comment]
please review

Max Lv <notifications@github.com>于2017年3月15日周三 22:40写道：

**Pull request merged**